### DISCUSSION TITLE
qalculate-gtk shows as black window on Monterey

### DISCUSSION THREAD

**Discussion by b-fitzpatrick** [Homebrew/discussions#2424] [Everyday usage]
After upgrading a 2015 MacBook Pro 15 to Monterey, qalculate-gtk shows as a black window. The application is running, as I was able to blindly type an expression and copy it to the clipboard. I've tried uninstalling and reinstalling gtk+3 and calculate-gtk.

I receive the following gtk warnings:
(qalculate-gtk:43615): Gtk-WARNING **: 08:20:39.108: Theme parsing error: <data>:1:16: Expected a string.
(qalculate-gtk:43615): Gtk-WARNING **: 08:20:39.109: Theme parsing error: <data>:1:16: Expected a string.
(qalculate-gtk:43615): Gtk-WARNING **: 08:20:39.109: Theme parsing error: <data>:1:28: Expected a string.
(qalculate-gtk:43615): Gtk-WARNING **: 08:20:39.109: Theme parsing error: <data>:1:16: Expected a string.
(qalculate-gtk:43615): Gtk-WARNING **: 08:20:39.109: Theme parsing error: <data>:1:16: Expected a string.

I don't know whether or not I received the same warnings on Big Sur, as I didn't launch it from the command line.

Any ideas?

**Comment by SMillerDev** [comment]
Why do you suspect this is a bug in the packaging of the software?

**Reply by b-fitzpatrick** [reply]
I don’t know what to suspect. I’m just hopeful that someone can give me some ideas to try to fix the issue.

**Reply by SMillerDev** [reply]
I think the best place for that would be the forum where the `qalculate-gtk` developers gather

**Reply by b-fitzpatrick** [reply]
I promise I had searched, but I didn't find this until now: https://gitlab.gnome.org/GNOME/gtk/-/issues/4395

It appears to have been fixed in gtk, but it hasn't been tagged yet.
